src/sentry/integrations/pagerduty/client.py

```diff
@@ -63,15 +63,23 @@ def send_trigger(self, data, organization=None, method="fire"):
             # the payload is for a metric alert
             payload = data
 
-        response = self.post("/", data=payload)
+        try:
+            response = self.post("/", data=payload)
+        except Exception as exc:
```

Comment:
Catching the exception and then immediately re-raising it is equivalent to not using a try/catch.